src/examples/lib/scanner.ts

```diff
@@ -7,7 +7,7 @@ import { BB9E } from '../../toys/bb9e';
 import { R2D2 } from '../../toys/r2d2';
 import { LightningMcQueen } from '../../toys/lightning-mcqueen';
 import { SpheroMini } from '../../toys/sphero-mini';
-import * as noble from 'noble';
+import noble from './noble-wrapper';
```

Comment:
I think we could directly import from `noble-mac` here because its a shim for `noble` on MacOS. But I didn't get `tslint` satisfied without real type definitions.

Reply:
I opened a pull request for `noble-mac` type definitions here: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/31786.
So we can get rid of the `noble-wrapper` when my pr is merged and the `@types/noble-mac` package will be published.

Reply:
My PR (DefinitelyTyped/DefinitelyTyped#31786) was merged, so I added @types/noble-mac as dev dependency and removed the dummy type declarations.